documentation/commands/git-ship.md

```diff
@@ -20,6 +20,7 @@ into the main branch, leading to linear history on the main branch.
 * pulls remote updates for `<branch_name>`
 * merges the main branch into `<branch_name>`
 * squash-merges `<branch_name>` into the main branch
+* opens an editor for the user to enter the commit message for the squash merge
```

Comment:
since the default message is now commented out, you are no longer *customizing* it but are required to enter the message if not provided by a command line argument.